ui/.yarn/releases/yarn-1.22.19.js

```diff
@@ -71633,9 +71633,9 @@ declare var __webpack_require__: mixed;
 
```

Comment:
According to the [yarn docs,](https://yarnpkg.com/getting-started/qa) this directory should actually be ignored 🤔

Reply:
ugh yeah... I noticed that. why the heck did this happen then?! 🤔